docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -795,6 +785,8 @@ We'll take care of provisioning a new repository and set it up to:
 - Configure your extension to use the link:https://github.com/quarkusio/quarkus-ecosystem-ci#quarkus-ecosystem-ci[Quarkus Ecosystem CI] to build against the latest Quarkus Core changes;
 - Give you the freedom to manage the project and release to Maven Central as you like.
 
+NOTE: Extensions hosted in the Quarkiverse Hub may or may not end up in the Quarkus platform.
```

Comment:
Is there any process someone could follow to have a Quarkiverse extension added to the platform?

Reply:
This would have to be discussed with the platform maintainers. Specifically, the extension should fit the primary purposes/use-cases the platform is targeting and match the quality requirements, i.e. be supersonic, subatomic and work in native.

Reply:
Right, just wondering if that would be something that readers might ask frequently, maybe we should elaborate more this note at some point (doesn't need to be in this PR, ofc)